src/appShell/App/footer.scss

```diff
@@ -25,6 +25,10 @@
             color: #666 !important;
             list-style: none;
             padding-left: 0;
+            display: flex;
+            flex-direction: column;
+            gap: 6px;
+            margin-top: 15px;
```

Comment:
I like the updated style for the header, but the updated footer looks a little bit too spread out. I still like the previous footer styles. But I will reserve my opinion and wait for @alisman to comment on this one.

Reply:
Thanks for the update! I will ping @alisman today to get his opinion on this.